client_plugin/vmdk_plugin/main.go

```diff
@@ -31,6 +31,11 @@ import (
 // main for docker-volume-vsphere
 // Parses flags, initializes and mounts refcounters and finally initializes the server.
```

Comment:
The code which parses flags is moved to startPluginServer. So should move this comment to that function.